policy-build-rust/action.yaml

```diff
@@ -35,3 +35,55 @@ runs:
       shell: bash
       run: |
         kwctl annotate -m ${{ inputs.metadata-file }} -o annotated-policy.wasm ${{ inputs.input-wasm }}
+    -
+      name: Generate the SBOM files
+      shell: bash
+      run: |
+        curl -Lo $RUNNER_TEMP/sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
```

Comment:
let's be explicit about the version we're going to use

Reply:
I've changed the tool used to generate the SPDX files and already define the version used. Thus, I'm closing this thread.